gcloud/bigtable/table.py

```diff
@@ -139,6 +139,36 @@ def create(self, initial_split_keys=None):
         # We expect a `._generated.bigtable_table_data_pb2.Table`
         client._table_stub.CreateTable(request_pb, client.timeout_seconds)
 
+    def rename(self, new_table_id):
+        """Rename this table.
+
+        .. note::
+
+            This cannot be used to move tables between clusters,
+            zones, or projects.
+
+        .. note::
+
+            The Bigtable Table Admin API currently returns
+
+                ``BigtableTableService.RenameTable is not yet implemented``
+
+            when this method is used. It's unclear when this method will
+            actually be supported by the API.
```

Comment:
So is that the intended behavior here? I've gotten a `NetworkError` for using 32 bytes in a field that was capped at 30 and gotten a `LocalError` for sending a fully valid request to a method that isn't implemented (though the proto doesn't mention this).